### DISCUSSION TITLE
revit export .i.bim

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#209] [Q&A] [answered]
bentley i-model plug-in for revit 10.0.0.24
reivt2020
revit export  use menu->bentley -> publish->select save type Bentley Mobile i-model File (*.model)
  sometime  rvt file can not export  any file  ,and show  Notification dialog "No i-model has been produced."  
who can help me.

**Comment by jffmarker** [comment, marked as answer]
I would recommend using the [iTwin Synchronizer](https://www.bentley.com/en/products/product-line/digital-twins/itwin-synchronizer) to create iModels. It supports many formats, including Revit, and will upload your changes to the iModel Hub for easy use with iModel.js.
